src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -637,3 +632,40 @@ export function createAnimatedComponent(
     );
   });
 }
+
+// This is a makeshift solution to handle both 0.73 and 0.74 versions of React Native.
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let getViewInfo = (element: any) => {
+  if (element._nativeTag) {
+    getViewInfo = getViewInfo73;
+    return getViewInfo73(element);
+  } else if (element.__nativeTag) {
+    getViewInfo = getViewInfoLatest;
+    return getViewInfoLatest(element);
+  }
+  return getViewInfo73(element);
+};
```

Comment:
Would it be a good idea to move those functions to a different file?

Reply:
sounds like a good idea